src/cms/views/dashboard/admin_dashboard_view.py

```diff
@@ -34,6 +37,18 @@ def get(self, request, *args, **kwargs):
         :return: The rendered template response
         :rtype: ~django.template.response.TemplateResponse
         """
+        all_feedback = Feedback.objects.filter(is_technical=True)[:5]
```

Comment:
The `is_technical=True` is meant for "admins" (us), not content managers/creators. I think this needs to change to `is_technical=False`

Reply:
Just realized that this is only the admin dashboard.

Reply:
Since this is the admin_dashboard (the landing page for admins and Integreat staff), I think it's fine to show the technical feedback here. But probably it makes sense to copy the overview also to the region dashboard (ah, just noticed that you wrote the same comment :slightly_smiling_face:)